koop_form/apps/form/apps.py

```diff
@@ -1,12 +1,16 @@
 from django.apps import AppConfig
-from django.core.management import call_command
+from django.db.models.signals import post_migrate
+
 
 class FormConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "apps.form"
 
     def ready(self):
         from . import signals
- #       call_command("crontab", "add")
+        from apps.form.signals import init_weight_scheme_with_zero
+        post_migrate.connect(init_weight_scheme_with_zero, sender=self)
```

Comment:
1. powinieneś móc użyć signals.init_weight_scheme_with_zero, bo masz już signals w namespacesie <br>
2. co się dzieje z crontab i add?

Reply:
crontab i add było zawsze wykomentowane. To zadanie jest realizowane w start.sh tam ten corntab jest inicjalizowany

Reply:
done